packages/translations/package.json

```diff
@@ -40,9 +40,10 @@
     "build": "yarn build:clean && tsc && tsc -p tsconfig.esm.json",
     "build:clean": "rimraf dist esm",
     "preversion": "yarn build",
-    "prepublishOnly": "yarn build",
```

Comment:
Removing this may result in an error when publishing. I would recommend to keep it.

Reply:
As far as I know, lerna handles the publishing and uses `yarn build`

Reply:
if i search for "prepublishOnly" in the codebase it's not used anywhere

Reply:
It's a standard npm script: https://docs.npmjs.com/cli/v8/using-npm/scripts

It's run before `npm publish`. A little history:

> Since npm@1.1.71, the npm CLI has run the prepublish script for both npm publish and npm install, because it's a convenient way to prepare a package for use (some common use cases are described in the section below). It has also turned out to be, in practice, [very confusing](https://github.com/npm/npm/issues/10074). As of npm@4.0.0, a new event has been introduced, prepare, that preserves this existing behavior. A new event, prepublishOnly has been added as a transitional strategy to allow users to avoid the confusing behavior of existing npm versions and only run on npm publish (for instance, running the tests one last time to ensure they're in good shape).

Reply:
alright, it's back :-)